site/PRD.md

```diff
@@ -0,0 +1,40 @@
+## NYC's Art Museum introduction
+
+New York has a large number of art galleries throughout the city, but many of them cannot meet the aesthetic requirements and travel expectations of visitors. Sometimes, certain selection needs to be made according to the characteristics and geographical characteristics of the art galleries.
+
+
+
+## data
+
+Open data from New York City and art gallery websites. At the same time, you need to locate the real-time location of the user, so you need map data.
+
+
+
+## Main display function
+
+The map shows the location of all the art galleries in New York City, all of which have the same map markings;
+
+People can search for the gallery by its name, and the website will display various information about the gallery:
+
+The exact address;
+
+Telephone;
+
+Postcode;
+
+Museum website;
+
+Museum display information;
+
+Gallery scale.
+
+According to this information, people can judge whether they are interested in the museum or not. If they are interested, they can say that they want to go, if they are not interested, they can choose not to go.
+
+
+
+## Punch in function
+
+In the comments section, you can record your browsing experience, or you can rate the gallery and save it.
+
+The galleries you've been to show different markers on the map.
```

Comment:
If you're going to use something like Firebase for this, it would be cool to use the [user sign-in features](https://firebase.google.com/docs/auth/web/firebaseui) to keep peoples visits separate (though that's definitely something we haven't done in class, so could be a stretch goal).